### PR TITLE
Added learn icon code

### PR DESCRIPTION
**Pull request by grliszas14**

### CODE REVIEW

**Review by igorkorsukov — APPROVED** [review]

File: src/framework/ui/view/iconcodes.h [hunk 1]
```diff
@@ -384,6 +384,7 @@ class IconCode
 
         PLUGIN = 0xF440,
         LYRICS = 0xF441,
+        LEARN = 0xF441,
```
Comment (RomanPudashkin) [inline src/framework/ui/view/iconcodes.h:387]: It's the same as the previous one. Is that intentional?

Comment (grliszas14) [reply src/framework/ui/view/iconcodes.h:387]: Yes, that's intentional, change suggested by @igorkorsukov

Comment (igorkorsukov) [reply src/framework/ui/view/iconcodes.h:387]: > It's the same as the previous one. Is that intentional?

Used for different purposes, but now has the same value (icon)

Comment (igorkorsukov) [reply src/framework/ui/view/iconcodes.h:387]: But if we want to change the  image of the “learn” icon, we will have to enter a new one and look for all the places in the source code where it is used

**Pull request merged**